docs/reference/tools-reference/llm-tool.md

```diff
@@ -83,3 +83,71 @@ Setup connections to provisioned resources in prompt flow.
 1. Setup and select the connections to OpenAI resources
 2. Configure LLM model api and its parameters
 3. Prepare the Prompt with [guidance](./prompt-tool.md#how-to-write-prompt).
+
```

Comment:
why not put this section into prompt-tool.md ?

Reply:
First, the error user met https://github.com/microsoft/promptflow/blob/a5e1ce4ada15b9375e623e0e9e128231737bc098/src/promptflow-tools/promptflow/tools/common.py#L59 is thrown from llm tool.

Second, parsing chat logic occurs in llm tool logic, not prompt tool. Prompt tool just show rendered jinja result, but we need parsed messages. It is weird to show in prompt tool.

Third, prompt tool may be less used since custom_llm type tool is supported.

Reply:
make sense